template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPage/.template.config/template.json

```diff
@@ -0,0 +1,36 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "shortName": "page",
+  "sourceName": "Index",
+  "primaryOutputs": [ { "path": "Index.cshtml"}, { "path": "Index.cs" } ],
```

Comment:
@rynowak - for adding conditions to primary outputs, the format is like this:
```
"primaryOutputs": [ 
  {  "path": "Index.cshtml" }, 
  { 
    "path": "Index.cs",
    "condition": "(!no-pagemodel)"
  } 
]
```
If I understood correctly, that example should do what we talked about earlier.